packages/extension/src/newtab/LinksForm.tsx

```diff
@@ -24,6 +24,12 @@ export function LinksForm({
     setValidInputs((state) => ({ ...state, [i]: isValid }));
   };
 
+  const handleInputChange = (i: number, value: string) => {
+    // Prepend "http://" if the value does not start with "http://" or "https://"
+    const updatedValue = value.startsWith('http://') || value.startsWith('https://') ? value : `http://${value}`;
```

Comment:
Thanks for the suggestion, great idea! 🚀 

Why not using `https` as default? _The safer the better_, not?